app/instances.go

```diff
@@ -73,42 +73,42 @@ func (a *App) createTriggers(tConfigs []*trigger.Config, runner action.Runner) (
 		}
 
 		if tConfig.Ref == "" && tConfig.Type != "" {
-			log.RootLogger().Warnf("trigger configuration 'type' deprecated, use 'ref' in the future")
+			log.RootLogger().Warnf("trigger [%s]'s configuration 'type' deprecated, use 'ref' in the future", tConfig.Id)
```

Comment:
log.RootLogger().Warnf("trigger [%s]'s configuration uses deprecated property 'type', use 'ref' in the future", tConfig.Id)